pkg/cmd/runsGet.go

```diff
@@ -37,7 +37,7 @@ var (
 func init() {
 	runsGetCmd.PersistentFlags().StringVar(&runName, "name", "", "the name of the test run we want information about")
 	runsGetCmd.PersistentFlags().StringVar(&age, "age", "", "the age of the test run(s) we want information about. Supported formats are: 'FROM' or 'FROM:TO', where FROM and TO are each ages,"+
-		" made up of an integer and a time-unit qualifier. Supported time-units are days('d'), weeks('w') and hours ('h'). If missing, the TO part is defaulted to '0h'. Examples: '--age 1d' , '--age 6h:1h' ")
+		" made up of an integer and a time-unit qualifier. Supported time-units are days('d'), weeks('w'), hours('h') and minutes('m'). If missing, the TO part is defaulted to '0h'. Examples: '--age 1d' , '--age 6h:1h' ")
```

Comment:
Can we call the function which builds the parameter list from here too instead of listing them all explicitly ?
eg: 
```
runsGetCmd.PersistentFlags().StringVar(&age, "age", "", "the age of the test run(s) we want information about. Supported formats are: 'FROM' or 'FROM:TO', where FROM and TO are each ages,"+
		" made up of an integer and a time-unit qualifier. Supported time-units are "+getTimeUnitListForMessage()+...
		```
?